sdcm/mgmt.py

```diff
@@ -243,6 +243,15 @@ def detailed_progress(self):
         relevant_key = [key for key in parsed_progress_table.keys() if parsed_progress_table[key]][0]
         return parsed_progress_table[relevant_key]
 
+    def wait_for_task_success_status(self, timeout=10800):
+        start_time = time.time()
+        while start_time + timeout > time.time():
+            if 'ERROR (4/4)' in self.manager_node.remoter.run(f'sctool -c {self.cluster_id} task list | grep {self.id}').stdout.strip():
```

Comment:
optionally it may be useful to print some log warning in case of an error that is not the 4th one, e.g. 'ERROR (2/4)'

Reply:
i don't think it is necessary, as it doesn't make a difference in the final result, and somewhere in the logs it will show there were those failures.